configparser.y

```diff
@@ -796,6 +853,17 @@ pattern_or_zone_option:
         yyerror("address range used for request-xfr");
       append_acl(&cfg_parser->pattern->request_xfr, acl);
     }
+  | VAR_REQUEST_XFR VAR_AXFR STRING STRING STRING
+    {
+      acl_options_type *acl = parse_acl_info(cfg_parser->opt->region, $3, $4);
+      acl->use_axfr_only = 1;
+      if(acl->blocked)
+        yyerror("blocked address used for request-xfr");
+      if(acl->rangetype != acl_range_single)
+        yyerror("address range used for request-xfr");
+      acl->tls_auth_name = region_strdup(cfg_parser->opt->region, $4);
```

Comment:
I think this should be `$5` instead of referencing the acl info key name.

Reply:
Thanks for spotting this!